### PR TITLE
Add remediations lab content

### PR DESCRIPTION
**Pull request by JAORMX**
This adds a section to the lab to work with remediations.

### CODE REVIEW

**Comment by JAORMX** [comment]
Pull-request updated, HEAD is now 3154da6d625ba03178dfb1e462ae554bec4fcd01

File: workshop/content/exercises/04-remediations.md [hunk 1]
```diff
@@ -1,9 +1,391 @@
...
+              filesystem: root
+              mode: 420
+              path: /etc/sysctl.d/75-sysctl_kernel_dmesg_restrict.conf
+  outdated: {}
+status:
+  applicationState: NotApplied
+```
+
+You'll notice that this remediation merely creates a MachineConfig object that sets the `dmesg_restrict` 
+sysctl. You'll also notice that there is a controller set in the `ownerReferences` key in the metadata
```
Comment (jhrozek) [inline workshop/content/exercises/04-remediations.md:92]: Is the owner object commonly called a controller?

File: workshop/content/exercises/04-remediations.md [hunk 2]
```diff
@@ -1,9 +1,391 @@
...
+Remediations are an important part of the compliance process. They fill in the technical gaps in your
+deployment, and when addressed, get you closer to compliance with the benchmark you need to comply to.
+
+Exploring the remediations
+--------------------------
+
+As you noticed in the previous section, the compliance operator already generated some remediations
+for you. They are accessible as any other Kubernetes object.
+
+You can even inspect them as other objects via `oc explain complianceremediation`. e.g. To see
```
Comment (jhrozek) [inline workshop/content/exercises/04-remediations.md]: lower-case "To"

File: workshop/content/exercises/04-remediations.md [hunk 3]
```diff
@@ -1,9 +1,391 @@
...
+
+Applying a remediation
+----------------------
+
+Now that we know what the remediation does and why it was suggested, let's apply it!
+
+Applying a remediation can be done by the admin manually. This means that you would need to download
+the remediation object and apply the object.
+
+however, it's also possible to let the compliance-operator do this for you. This is what the `apply` flag
```
Comment (jhrozek) [inline workshop/content/exercises/04-remediations.md]: Capitalize H in however

File: workshop/content/exercises/04-remediations.md [hunk 4]
```diff
@@ -1,9 +1,391 @@
...
+Now we can just let the **machine-config-operator** do its job and apply this configuration
+to all of the hosts whose **MachineConfigPool** matches the `worker` role.
+
+You might have noticed that the **machine-config-operator** did a rolling restart of the nodes in
+order to apply the configuration. This is normal and expected. However, this might be problematic
+if you're applying several MachineConfig object changes, as each change will trigger a restart of
+all the nodes in that pool.
+
+Let's wait for the `worker` **MachineConfigPool** to finish updating before continuing to the next
+section.
```
Comment (jhrozek) [inline workshop/content/exercises/04-remediations.md]: We might want to suggest how, e.g. `oc get mcp -w` or even `watch oc get mcp`

File: workshop/content/exercises/04-remediations.md [hunk 5]
```diff
@@ -1,9 +1,391 @@
...
+away thanks to the `roles` key in the **ScanSettings** object.
+
+Letting the operator apply the remediations
+-------------------------------------------
+
+Applying each scan can be fairly tedious. This is why the compliance-operator has a flag for this!
+
+
+Applying scans is done in a suite level, which means it's done for all the scans that belong to the 
+benchmark you're currently scanning.
```
Comment (jhrozek) [inline workshop/content/exercises/04-remediations.md]: Should we suggest only doing this once the remediations have been reviewed or audited?

**Review by jhrozek — COMMENTED** [review]
Looks good, some comments inline.

File: workshop/content/exercises/04-remediations.md (hunk 1 above)
Comment (JAORMX) [reply workshop/content/exercises/04-remediations.md:92]: not always. The `controller` field mainly indicates that there is an object whose controller owns the object. We seem to be setting this value, hence why I mentioned it.

File: workshop/content/exercises/04-remediations.md (hunk 5 above)
Comment (JAORMX) [reply workshop/content/exercises/04-remediations.md]: Done.

**Comment by JAORMX** [comment]
Pull-request updated, HEAD is now 08b21262ebd04f6db960430f58c8130d9f40b9c9

File: workshop/content/exercises/04-remediations.md (hunk 4 above)
Comment (JAORMX) [reply workshop/content/exercises/04-remediations.md]: Done.

File: workshop/content/exercises/04-remediations.md (hunk 3 above)
Comment (JAORMX) [reply workshop/content/exercises/04-remediations.md]: Done.

File: workshop/content/exercises/04-remediations.md (hunk 2 above)
Comment (JAORMX) [reply workshop/content/exercises/04-remediations.md]: done.

**Review by jhrozek — APPROVED** [review]

**Pull request merged**